Alignment/OfflineValidation/python/TkAlAllInOneTool/genericValidation.py

```diff
@@ -281,10 +281,13 @@ def __init__(self, valName, alignment, config):
 
         # if maxevents is not specified, cannot calculate number of events for
         # each parallel job, and therefore running only a single job
-        if int( self.general["maxevents"] ) == -1 and self.NJobs > 1:
+        if int( self.general["maxevents"] ) < 0 and self.NJobs > 1:
             msg = ("Maximum number of events (maxevents) not specified: "
                    "cannot use parallel jobs.")
             raise AllInOneError(msg)
+        if int( self.general["maxevents"] ) / self.NJobs != float( self.general["maxevents"] ) / self.NJobs:
+            msg = ("maxevents has to be divisible by parallelJobs")
+            raise AllInOneError(msg)
```

Comment:
@hroskes isn't that too restrictive as we could just take the remainder of tracks from the last job?

Reply:
I think it's easier this way, if you have 100k events and you want to have 3 jobs, you can just make it 99999 and you won't see the difference.  The previous behavior (for maxevents) was to just silently change it to 100000 / 3 = 33333 for each job.  At least this way you know exactly what's happening.

Reply:
ok